src/main/java/htsjdk/samtools/DuplicateScoringStrategy.java

```diff
@@ -80,6 +81,8 @@ public static short computeDuplicateScore(final SAMRecord record, final ScoringS
                         score += SAMUtils.getMateCigar(record).getReferenceLength();
                     }
                     break;
+                case RANDOM:
+                    score += (short) Math.sqrt(record.getReadName().hashCode());
```

Comment:
Since we already have the murmur3 hash in htsjdk, and given that we already agreed for other parts of the code that it is better than the default String.hashCode implementation, could you use murmur3 instead?